data/lib/core/player.lua

```diff
@@ -358,3 +358,19 @@ end
 function Player.getTotalMoney(self)
 	return self:getMoney() + self:getBankBalance()
 end
+
+function Player.addAllAddons(self, addon)
```

Comment:
It sounds quite confusing when you're adding "all addons" but... need to provide addon as argument, no?

Maybe `addAddonToAllOutfits` is what it actually does?

Reply:
yes, it add addon set 1 or 2, too all outfits